parquet-column/src/test/java/org/apache/parquet/io/ExpectationValidatingConverter.java

```diff
@@ -151,14 +151,16 @@ public Void getCurrentRecord() {
   }
 
   private String path(List<GroupType> path, Type type) {
-    String pathString = "";
+    StringBuilder pathString = new StringBuilder();
     if (path.size() > 0) {
       for (int i = 1; i < path.size(); i++) {
-        pathString += path.get(i).getName() + ".";
+        pathString.append(path.get(i).getName());
```

Comment:
nit: you may keep the chain pattern as `append` returns the builder object.
Do not have a strong opinion on the current or the chained version.